src/main.js

```diff
@@ -224,21 +224,27 @@ function initLocales() {
 
     let defaultLang = state.setting('language');
     let preferredLangs = (window.navigator && window.navigator.languages) || [];
-    let preferredLang = preferredLangs[0];
 
+    // on chrome navigator.language is the default language not the first in the list
+    if (window.navigator && window.navigator.language) {
+        preferredLangs = _.merge([window.navigator.language], preferredLangs);
+    }
+    // our configs default lang overrides all others
     if (defaultLang) {
-        i18next.changeLanguage(defaultLang, (err, t) => {
-            if (err) {
-                i18next.changeLanguage('en-us');
-            }
-        });
-    } else if (preferredLang) {
-        i18next.changeLanguage(preferredLang, (err, t) => {
-            if (err) {
-                i18next.changeLanguage('en-us');
-            }
-        });
+        preferredLangs = _.merge([defaultLang], preferredLangs);
     }
+
+    _.forEach(preferredLangs, (lang) => {
+        if (_.includes(AvailableLocales.locales, lang.toLowerCase())) {
+            i18next.changeLanguage(lang, (err, t) => {
+                if (err) {
+                    i18next.changeLanguage('en-us');
```

Comment:
This returns before your loop has finished, it needs to give a chance for all languages to match before you fall back to default.
Other approach, set this by default and override with the loop.